nncf/quantization/algo.py

```diff
@@ -994,6 +1001,40 @@ def _add_single_weight_quantizer(self, target_model: NNCFNetwork, insertion_poin
         command = InsertionCommand(insertion_point, op, OperationPriority.QUANTIZATION_PRIORITY)
         return quantizer_id, command
 
+    def _get_adjust_padding_args(self, quantization_point: SingleConfigQuantizationPoint,
+                                 quantizer_module_id: NonWeightQuantizerId,
+                                 target_model: NNCFNetwork,
+                                 quantization_points: List[SingleConfigQuantizationPoint]):
+        result = []
+        if quantization_point.is_adjust_padding_applicable:
```

Comment:
As discussed, should remove the `is_adjust_padding_applicable` from quantization points and instead determine a matching between a weight-quantized module and its associated activation quantizer from the rest of the quantizer setup.

Reply:
Done